File: Test/src/T6.java
import java.util.Arrays;
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class T6 {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		String str="";
		String[] result;
		if(args.length>0)
		{
			for (String s : args) 
			{
				str=str+s+" ";
			}
			
		}
		else
		{
			
			str = in.nextLine();
		}
		result=str.split(" ");
		Arrays.sort(result);
		Map<String,Integer> word_count = new HashMap<String,Integer >();
		for(String i:result)
		{
			//for(Object)
			if(word_count.containsKey(i))
				{int temp=word_count.get(i);
				 temp++;
				 word_count.put(i, temp);
				}
			else
				
				word_count.put(i,1);
			
		}
		for(Map.Entry<String,Integer > item : word_count.entrySet()){
	        
	           System.out.printf("%s %d \n", item.getKey(), item.getValue());
	       }
		
	}

}

File: Test/T6.kt
import java.util.*
fun main(args: Array<String>) {
val scan = Scanner(System.`in`)
   var str=emptyArray<String>()
  	var a=emptyArray<words>()
  	val MapStr= mutableMapOf("" to 0)

  	if((args.size)>0)
		{
			for ( s in args) 
			{
				str=str+s;
			}
			
		}
		else
		{
			
			var temp=scan.nextLine()
			val temp2=temp.split(" ")
			for ( s in temp2) 
			{
				str+=s
			}
		}
		
	str.sort()
	for (i in str) 
	{
   		if(MapStr.containsKey(i))
   			MapStr+=i to MapStr.getValue(i)+1
   		else
   			MapStr.put(i,1)
	}	
	MapStr.remove("")
	for (i in MapStr)
		a+=words(i.key,i.value)
	for ( r in 1..(a.size-1)) 
		{
			for ( j in 0..(a.size-1-r) )
				if(a[j].count < a[j+1].count)
				{
					// Обмен местами
					var temp=a[j]
					a[j] = a[j+1];
					a[j+1] = temp;
				}

		}
	for ( i in a) 
		println("${i.word} ${i.count}")

}	
class words(w:String,c:Int)
{
	
	var word:String
	var count:Int
	init{
		word=w
		count=c
	}


}
